.github/workflows/benchmark.yml

```diff
@@ -103,33 +115,57 @@ jobs:
     permissions:
       pull-requests: write
       contents: write
+    env:
+      BENCHER_PROJECT: tailcall
+      BENCHER_ADAPTER: rust_criterion
+      BENCHER_TESTBED: benchmarking-runner
+      PR_BENCHMARK_RESULTS: benches/change.txt
+      BASE_BENCHMARK_RESULTS: benches/base.txt
+      BRANCH_NAME: ${{ github.head_ref }}_commit_${{ github.event.pull_request.head.sha }}
```

Comment:
Creating a new branch name yourself should no longer be necessary, see: https://bencher.dev/docs/how-to/track-benchmarks/#relative-continuous-benchmarking